Sources/PointFree/Admin/Index.swift

```diff
@@ -0,0 +1,138 @@
+import Css
+import Either
+import Foundation
+import Html
+import HtmlCssSupport
+import HttpPipeline
+import HttpPipelineHtmlSupport
+import Optics
+import Prelude
+import Styleguide
+import Tuple
+
+private let adminEmails = [
```

Comment:
this file contains the beginning of some admin pages. it's all very basic, but I added it cause I wanted a page where we could send emails for new episodes easily.